updatecli/updatecli.d/terraform-providers.yaml

```diff
@@ -27,12 +27,11 @@ targets:
       file: ".terraform.lock.hcl"
     scmID: default
 
-pullrequests:
+actions:
   default:
-    kind: github
-    scmID: default
-    targets:
-      - upgradeActualLockFile
+    kind: github/pullrequest
+    scmid: default
     spec:
       labels:
         - dependencies
+        - terraform-providers
```

Comment:
```suggestion
        - terraform-providers

```
Missing EOL

Reply:
You can see the absence of End Of File in github diff:
<img width="147" alt="Screenshot 2023-06-23 at 15 25 49" src="https://github.com/jenkins-infra/digitalocean/assets/91831478/30f3b2ec-23ce-4033-a9c2-6a3d8a310070">

The other files have an EOL.